packages/flutter_tools/lib/src/drive/web_driver_service.dart

```diff
@@ -224,19 +223,42 @@ class WebDriverService extends DriverService {
 }
 
 /// A list of supported browsers.
-enum Browser {
-  /// Chrome on Android: https://developer.chrome.com/multidevice/android/overview
+enum Browser implements CliEnum {
+  /// Chrome on Android: https://developer.chrome.com/docs/multidevice/android/
   androidChrome,
+
   /// Chrome: https://www.google.com/chrome/
   chrome,
-  /// Edge: https://www.microsoft.com/en-us/windows/microsoft-edge
+
+  /// Edge: https://www.microsoft.com/edge
   edge,
+
   /// Firefox: https://www.mozilla.org/en-US/firefox/
   firefox,
+
   /// Safari in iOS: https://www.apple.com/safari/
   iosSafari,
+
   /// Safari in macOS: https://www.apple.com/safari/
-  safari,
+  safari;
+
+  @override
+  String get helpText => switch (this) {
+        Browser.androidChrome => 'Chrome on Android (see also "--android-emulator").',
+        Browser.chrome => 'Google Chrome on this computer (see also "--chrome-binary").',
+        Browser.edge => 'Microsoft Edge on this computer (Windows only).',
+        Browser.firefox => 'Mozilla Firefox on this computer.',
+        Browser.iosSafari => 'Apple Safari on an iOS device.',
+        Browser.safari => 'Apple Safari on this computer (macOS only).',
+      };
```

Comment:
```suggestion
    Browser.androidChrome => 'Chrome on Android (see also "--android-emulator").',
    Browser.chrome => 'Google Chrome on this computer (see also "--chrome-binary").',
    Browser.edge => 'Microsoft Edge on this computer (Windows only).',
    Browser.firefox => 'Mozilla Firefox on this computer.',
    Browser.iosSafari => 'Apple Safari on an iOS device.',
    Browser.safari => 'Apple Safari on this computer (macOS only).',
  };
```

Reply:
why fight the formatter? It just makes everyone work harder!

Reply:
why fight the flutter style guide on a tools PR?

Reply:
See previous comment. 😜